bin/configs/typescript-angular-v15-query-param-object-format.yaml

```diff
@@ -0,0 +1,8 @@
+generatorName: typescript-angular
```

Comment:
I suggest to remove this config, since this there are no angular 15 specific changes to the code